components/badge.tsx

```diff
@@ -1,22 +1,25 @@
+import { ReactElement } from "react";
 import { Badge as ReactBadge } from "react-bootstrap";
 
 interface BadgeProps {
-  label: string;
+  label?: string | ReactElement;
```

Comment:
I'm thinking you wait "either label or children" so this would be `label?: string` and not `string | ReactElement`